var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -40,6 +41,9 @@ def install(self, spec, prefix):
         if '+shared' in self.spec:
             configure_args.append("--enable-shared")
 
+        if not self.compiler.f77 and not self.compiler.fc:
+            configure_args.append('--enable-fortran=no')
```

Comment:
Without a `configure.ac`, the CMake build is probably closer to working on OS X than the `configure` script is…